src/Syntax/Expr.hs

```diff
@@ -234,7 +233,7 @@ isFn _ = False
 conVarRef :: Var a ~ Var Resolved => Expr a -> Bool
 conVarRef (VarRef t _) =
   case t of
-    TgName t _ -> isUpper (T.head t)
+    TgName t i -> i == (-36) || isUpper (T.head t) -- -36 is lAZYName
```

Comment:
We really need a better way of doing this, especially as constants are less fixed now.

Reply:
Thanks :)